src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -1300,6 +1300,7 @@ private bool ConsumeString()
             {
                 if (IsLastSpan)
                 {
+                    _bytePositionInLine += localBuffer.Length + 1;  // Account for the start quote
```

Comment:
The code above does
```cs
int idx = localBuffer.IndexOfQuoteOrAnyControlOrBackSlash();
```

where idx must be -1 for the code to get here. Doesn't that mean the quote wasn't found? i.e. why is +1 being added here? (I'm not familiar with the code)

Reply:
Yes, the "end quote" wasn't found. The + 1 is for the "start quote" of the JSON string. We know that the start quote exists, which is why we know to consume a JSON string.

Reply:
I was assuming this was start quote, so end quote makes sense. Thanks